_topic_maps/_topic_map_rosa.yml

```diff
@@ -236,11 +236,13 @@ Topics:
   File: rosa-sts-aws-prereqs
 - Name: ROSA IAM role resources
   File: rosa-sts-ocm-role
-- Name: Limits and scalability
-  File: rosa-limits-scalability
-- Name: ROSA with HCP limits and scalability
-  File: rosa-hcp-limits-scalability
-- Name: Planning resource usage in your cluster
+##### NOTE: THE BELOW IS REMOVED AS PART OF OSDOCS-13310
+# - Name: Limits and scalability
+#   File: rosa-limits-scalability
+#- Name: ROSA with HCP limits and scalability
```

Comment:
I see this assembly in [SA with HCP limits and scalabil](https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-hcp-limits-scalability) but this is OK?

Reply:
Should be omitted from both now